src/redux/auth-reducer.js

```diff
@@ -0,0 +1,131 @@
+import { userAPI } from '../api';
+
+const SET_AUTH_USER_DATA = 'SET_AUTH_USER_DATA';
+const SET_AUTH_USER_STATUS = 'SET_AUTH_USER_STATUS';
+const DELETE_AUTH_USER_DATA = 'DELETE_AUTH_USER_DATA';
+const CLEAR_AUTH_USER_STATUS = 'CLEAR_AUTH_USER_STATUS';
+const SET_USER_DATA = 'SET_USER_DATA';
+
+const initialState = {
+  nickname: null,
+  id: null,
+  email: null,
+  login: false,
```

Comment:
do you mean, that user is logged into the system?? 
if so, https://translate.google.com/#view=home&op=translate&sl=ro&tl=en&text=eu%20sunt%20logat
you better use: **loggedIn**
to make it even more useful -> you can call it - **isLoggedIn**

then you can use it: 
```
isLoggedIn ? renderLoggedInItems : renderLoggedOutItems
```